sklearn/inspection/_partial_dependence.py

```diff
@@ -532,64 +519,48 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
         Please see :ref:`this note <pdp_method_differences>` for
         differences between the 'brute' and 'recursion' method.
 
+        Note: Ignored by ICE calculation at the moment and 'brute' is used.
+
     n_jobs : int, optional (default=None)
         The number of CPUs to use to compute the partial dependences.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    verbose : int, optional (default=0)
-        Verbose output during PD computations.
+        verbose : int, optional (default=0)
```

Comment:
no need to indent

Reply:
Fixed in https://github.com/scikit-learn/scikit-learn/pull/16164/commits/4a31914f72a50286e2248465c0641e4e38d5b337